app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
```

Comment:
Using a class attribute `people` to store instances of `Person` can lead to issues if multiple `Person` objects are created in different contexts. Consider using an instance attribute or an external data structure to manage these instances.